org.eclipse.scanning.test/src/org/eclipse/scanning/test/malcolm/AbstractMultipleClientMalcolmTest.java

```diff
@@ -46,7 +46,7 @@ public void twoDevicesOneGettingState() throws Throwable {
 		// device connections (although MockService will not do this)
 		final Collection<DeviceState> states = new HashSet<DeviceState>();
 		try {
-			IMalcolmDevice zebra = service.getDevice("zebra");
+			IMalcolmDevice zebra = (IMalcolmDevice) service.getRunnableDevice("zebra");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "zebra" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)